R/appender.R

```diff
@@ -97,3 +109,29 @@ appender.tee <- function(file){
     cat(line, file=file, append=TRUE, sep='')
   }
 }
+
+# Write to a dynamically-named file (and optionally the console)
+appender.file2 <- function(format, console = FALSE,
+                           default.func = "shell", datetime.fmt = "%Y%m%dT%H%M%S"){
```

Comment:
- Not certain what you mean by *"use two spaces for function line wrapping"*, sorry, can you explain please?
- `default.func` : the intent was to allow the user to define what the "top-level function" name should be, and it's hard-coded in `.get.parent.func.name`. If a user wants to override what the top-level is represented as. It should really be an option tied back to `.get.parent.func.name` instead. I'll take it out and let you mull it over.